docs/developers/data/abi.md

```diff
@@ -279,3 +301,225 @@ Similarly, [enums](/developers/data/custom-types#custom-enums) are defined by:
         - Struct-like variants, with named fields.
 
 You can read more about Rust enums [here](https://doc.rust-lang.org/book/ch06-01-defining-an-enum.html).
+
+---
+
+[comment]: # (mx-context-auto)
+
+## ESDT Attribute ABI
+
+### Overview
+Even though the `#[derive(TypeAbi)]` annotation exports all the data types used in the contract along with the type descriptions, we have also implemented a manual way to export data about an ESDT. The structure resembles an ESDT ticker and its attributes' type and gets exported in the main ABI file (and also in separate json files) in order to fetch it easier from other services (e.g. the developer can define the type of the attributes expected for a specific ESDT, fetch it from abi and use it easily in the frontend for a more specific result).
+
+Starting with the framework version `0.44`, developers can use the new trait annotation `#[esdt_attribute("name", Type)]` in order to export ESDT attributes types in the ABI file. 
+
```

Comment:
```suggestion
The name field is an arbitrary name provided by the developer, to identify the token. Token identifiers are not hard-coded in contracts, but it can make sense to use the ticker here, if known.

The type field is simply the name of the type, as it would show up in regular smart contract code.
```